airfoil/lib/airfoil.rb

```diff
@@ -26,15 +21,17 @@ def create_stack
         puts "Received SIGTERM, shutting down gracefully..." # rubocop:disable Rails/Output
       end
 
+      logger = defined?(::Rails) ? Rails.logger : Logger.new($stdout, level: (ENV["LOG_LEVEL"] || :info).to_sym)
+
       ::Middleware::Builder.new { |b|
-        b.use Middleware::LoggerTagging, Rails.logger
+        if defined?(::Rails)
+          b.use Middleware::LoggerTagging, logger
```

Comment:
With logger defined above, do we still want on use it here only if running within Rails?

Reply:
The tagged logging functionality is Rails-specific (https://api.rubyonrails.org/classes/ActiveSupport/TaggedLogging.html#method-i-tagged) - we could make the middleware optional I suppose, and that way apps not using Rails but using Ruby with some other configuration for a global logger could still use this bit?

Reply:
FYI, the Datadog logger requires some version of a tagged logger. Not sure if this will impact or not.